Automattic/jetpack
# Related posts block: add srcset to thumbnails

## Proposed changes:

In an effort to move away from the `devicepx` script towards native functionality, this PR adds a `srcset` to related posts thumbnails (the block, not the module feature). The browser will automatically pick the most appropriate image from the options in the `srcset`, depending on the screen pixel ratio and the current zoom value.

* Add `srcset` to the related posts block thumbnails
* Add 1.5x multiplier, because this is an extremely common screen density

This PR complements #31432, by dealing with the related posts block instead of the module feature, this time.

### Other information:

- N/A Have you written new tests for your changes, if applicable?

## Jetpack product discussion
None.

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
- Add a "Related Posts" block to your test post, and ensure that it can find at least 3 related posts.
- Enable thumbnails in the block options.
- Visit the post.
- Verify that the thumbnails include a `srcset` with different pixel ratios (up to 4x, if the source image is large enough).
